docs(editor): state machine layers (#754)
rive-app/rive-docs: editor/state-machine/layers.mdx

<YouTube id="7Vb1LosMzwk" />

A state machine layer can play one state at a time. Add multiple layers when you want a state machine to play multiple animations or interactions at once.

For example, one layer might control a character's idle and walk animations, while another layer controls a separate hover or selected state.

## Creating a new layer

To create a layer, click the **+** button in the **Layers** tab.

![Adding a new layer](/images/editor/state-machine/state-machine-layer-create.gif)

## Layer Order

Layer order determines which layer takes priority when multiple layers animate the same property.

> Layers farther to the right take priority over layers to the left. In most cases, this does not matter. However, if multiple layers control the same object property, the animation in the rightmost layer takes priority as the layers mix.

Reviewer: ```suggestion
Layers farther down the list take priority over layers above them. In most cases, this does not matter. However, if multiple layers control the same object property, the animation in the lower layer takes priority as the layers mix.
```
